src/OpenTelemetry/Trace/Export/SimpleActivityProcessor.cs

```diff
@@ -0,0 +1,103 @@
+﻿// <copyright file="SimpleActivityProcessor.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Trace.Export
+{
+    /// <summary>
+    /// Implements simple activity processor that exports activities in OnEnd call without batching.
+    /// </summary>
+    public class SimpleActivityProcessor : ActivityProcessor, IDisposable
+    {
+        private readonly ActivityExporter exporter;
+        private bool disposed = false;
+        private long spanCount = 0;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SimpleActivityProcessor"/> class.
+        /// </summary>
+        /// <param name="exporter">Activity exporter instance.</param>
+        public SimpleActivityProcessor(ActivityExporter exporter)
+        {
+            this.exporter = exporter ?? throw new ArgumentNullException(nameof(exporter));
+        }
+
+        public long GetSpanCount()
```

Comment:
I'm on the fence - whether this should be called a span count or activity count.

Reply:
And do we really need it (considering we need atomic operation to maintain the counter).

Reply:
removed them all.